assets/i18n/en.json

```diff
@@ -289,6 +289,7 @@
     "I18N_LEARNER_DASHBOARD_EMPTY_FEEDBACK_THREADS": "It looks like you don't have any active feedback threads. Your feedback helps us improve the quality of our lessons. Head over to the library to learn something exciting and submit your valuable feedback!",
     "I18N_LEARNER_DASHBOARD_EMPTY_INCOMPLETE_COLLECTIONS": "It looks like you have no partially-complete collections at the moment. Head over to the library to start an exciting new collection!",
     "I18N_LEARNER_DASHBOARD_EMPTY_INCOMPLETE_EXPLORATIONS": "It looks like you have no partially-complete explorations at the moment. Head over to the library to start an exciting new exploration!",
+    "I18N_LEARNER_DASHBOARD_EMPTY_SKILL_PROFICIENCY": "It look like you have not started any topic yet. Head over to the <a href=\"http://localhost:8181/learn/math\" class=\"inline-links\">classroom</a> to start an exciting new topic!",
```

Comment:
Incorrect link. Just `/learn/math` should work, right?

Reply:
Just to check, were there existing cases where html was there in translation? If not, I'd suggest using variables here instead of html and the variable gets filled with html in the component (variables are used in some other translation keys as well).
Or even better, to make sure everything gets translated, even the link, split this text into 3 parts, text before link, text in link and text after link and use 3 translation keys which will make sure all the text gets translated. Having HTML in translation keys is generally not used, I believe.
It's also there in line 294 (which I had missed in my previous reviews, sorry about that). Make sure all cases where you have html in translation is replaced with this.

Reply:
Yeah, there were existing cases where Html was there in the translation (like line 599). But if we split this into 3 parts, just to confirm, the second part i.e. the link, will still be an HTML anchor tag right?

Reply:
Done. PTAL!